### PR TITLE
Feature : adapt to new core modules

### PR DESCRIPTION
**Pull request by engdorm**
updated to the latest code and docker images
contains the latest code from the backend and frontend

### CODE REVIEW

**Comment by engdorm** [comment]
@AlexFernandes-MOVAI Can you help with the CI 
something went wrong with the ros container

**Comment by MithunKinarullathil** [comment]
This releases/3.0.1 seems to be very old than the current main. Can you make a new releases branch from main and redo this PR? Syncing it would make another pre-release build I think, which we don't want.

File: docker-compose.yml [hunk 1]
```diff
@@ -151,7 +156,7 @@ services:
       APT_INSTALL_LIST: "$STUDIO_APP_PACKAGES"
       HTTP_PORT: ${STUDIO_HTTP_PORT}
       DEFAULT_USERNAME: movai
-      DEFAULT_PASSWORD: movai
+      DEFAULT_PASSWORD: www.mov.ai
```
Comment (AlexFernandes-MOVAI) [inline docker-compose.yml:159]: take care of adapting the sanity test:
https://github.com/MOV-AI/movai-flow/blob/7a5659fde16fcda36ff8b1569e226bfaf2ff017e/.github/workflows/TestOnPR.yml#L83

File: docker-compose.yml [hunk 2]
```diff
@@ -68,8 +68,11 @@ x-movai-common:
     IGN_PARTITION: ${IGN_PARTITION:?missing IGN_PARTITION ENV var}
     IGNITION_VERSION: fortress
     APT_AUTOINSTALL: "once"
-    APT_KEYS_URL_LIST: "https://artifacts.aws.cloud.mov.ai/repository/movai-applications/gpg"
-    APT_REPOS_LIST: "deb [arch=all] https://artifacts.aws.cloud.mov.ai/repository/ppa-public main main"
+    # APT_KEYS_URL_LIST: "https://artifacts.aws.cloud.mov.ai/repository/movai-applications/gpg"
+    # APT_REPOS_LIST: "deb [arch=all] https://artifacts.aws.cloud.mov.ai/repository/ppa-public main main"
+    # For DEV purpose:
```
Comment (AlexFernandes-MOVAI) [inline docker-compose.yml:73]: we should make sure that all packages are published to `ppa-public`

**Review by AlexFernandes-MOVAI — CHANGES_REQUESTED** [review]

**Comment by AlexFernandes-MOVAI** [comment]
will be addressed by #225

**Pull request closed without merging**